docs/commands.md

```diff
@@ -4,7 +4,9 @@ All build commands are executed via [NPM Scripts](https://docs.npmjs.com/misc/sc
 
 ### `npm run dev`
 
-> Starts a Node.js local development server. See [API Proxying During Development](proxy.md) for more details.
+Starts a Node.js local development server. See [API Proxying During Development](proxy.md) for more details.
+
+> Sets process.env.NODE_ENV to `'development'`
```

Comment:
I added these notes to that people get the connection between the commands and the NODE_ENV the set. These define the "mode" the app is running in.